docs/USAGE.md

```diff
@@ -58,7 +58,9 @@ Use these environment variables to configure the tracing library:
 | `OTEL_MAX_LOGFILE_SIZE` | The maximum size for tracer log files, in bytes. | `10 MB` |
 | `OTEL_TRACE_LOG_PATH` | The path of the profiler log file. | Linux: `/var/log/OTEL/dotnet/dotnet-profiler.log`<br>Windows: `%ProgramData%"\OTEL .NET Tracing\logs\dotnet-profiler.log` |
 | `OTEL_DIAGNOSTIC_SOURCE_ENABLED` | Enable to generate troubleshooting logs with the `System.Diagnostics.DiagnosticSource` class. | `true` |
-| `OTEL_DOTNET_TRACER_DISABLED_INSTRUMENTATIONS` | The instrumentations you want to disable, if any, separated by a comma. These are the supported integrations: `AspNet`, `HttpClient`, `SqlClient`, `MongoDb` |  |
+| `OTEL_INTEGRATIONS` | The file path of IL-rewrite instrumentations JSON configuration file. | `%ProfilerDirectory%/integrations.json` |
```

Comment:
The description covers it well but the name seems rather opaque. Not sure how to improve it, perhaps `OTEL_DOTNET_TRACER_IL_REWRITE_CONFIG_FILE`? It seems awfully long but conveys better what it actually it is. Any other ideas?

Reply:
I can do it in next PR 😉 
I just saw that we are missing docs for this env var.